apps/opik-frontend/Dockerfile

```diff
@@ -11,12 +11,29 @@ RUN npm install
 COPY . .
 
 ARG OPIK_VERSION
+ARG SENTRY_AUTH_TOKEN
```

Comment:
@liyaka 
is there a way to take this SENTRY_AUTH_TOKEN from `build_and_push_docker.yaml` securely? my dockerfile image logs were saying that we shouldn't keep sensitive info in `ARG` and `ENV` vars